### PR TITLE
Functions for User's authenticated by Email/Password combo

### PR DESCRIPTION
**Pull request by stevewpatterson**

### CODE REVIEW

**Comment by michaelspeed** [comment]
there could be one more function where users forgets password!

File: email-user-management/functions/signup/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+```sh
+graphcool playground
+```
+
+Run this mutation to create a user:
+
+```graphql
+mutation {
+  # replace __EMAIL__ and __PASSWORD__
+  createEmailUser(email: "__EMAIL__", password: "__PASSWORD__") {
```
Comment (stevewpatterson) [inline email-user-management/functions/signup/README.md]: @marktani after your changes I think this should be signupEmailUser

Comment (marktani) [inline email-user-management/functions/signup/README.md]: good catch! thanks 👍

**Comment by marktani** [comment]
Thanks so much @stevewpatterson this is awesome :tada:

**Pull request merged**